codespell_lib/data/dictionary.txt

```diff
@@ -28900,12 +28900,28 @@ iniate->initiate
 iniated->initiated
 iniates->initiates
 iniating->initiating
+iniation->initiation
```

Comment:
Any `iniation->indication` (and friends) typo?

Reply:
I couldn't find these typos.